hooks/checkout

```diff
@@ -38,7 +38,11 @@ checkout() {
   git fetch -v origin HEAD
   git config remote.origin.fetch
   git fetch -v --prune origin "${BUILDKITE_BRANCH}"
-  git checkout -f "${BUILDKITE_COMMIT}"
+  if [[ "${BUILDKITE_COMMIT}" == "HEAD" ]]; then
```

Comment:
Should we check that `BUILDKITE_COMMIT` isn't an empty string too?

Reply:
👌